src/AzureAuth/Program.cs

```diff
@@ -31,7 +31,8 @@ private static void Main(string[] args)
             // trigger telemetry before we ever get to disable it.
             //
             // To disable telemetry a user need only leave this environment variable unset. It's off by default.
-            string applicationInsightsIngestionToken = Environment.GetEnvironmentVariable(EnvVars.ApplicationInsightsIngestionTokenEnvVar);
+            // string applicationInsightsIngestionToken = Environment.GetEnvironmentVariable(EnvVars.ApplicationInsightsIngestionTokenEnvVar);
+            string applicationInsightsIngestionToken = "2ba47d2c-65c4-4bf9-a77a-f8ffd05cbafe";

```

Comment:
Is this an internal token? If yes, delete this branch as soon as possible. It's too dangerous.